vividus/src/test/java/org/vividus/configuration/VaultStoredPropertiesProcessorTests.java

```diff
@@ -156,4 +163,24 @@ void shouldUseNamespaceForEnterpriseVaultWhenConfigured() throws IOException
             ordered.verify(context).getBean(VaultTemplate.class);
         }
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"HASHI_CORP_VAULT()", "VAULT()"})
+    void shouldPrintLogAboutDeprecation(String expression) throws IOException
+    {
+        try (var processor = new VaultStoredPropertiesProcessor(new Properties()))
+        {
+            processor.processProperty("test-property-name", expression);
+        }
+        if ("VAULT()".equals(expression))
```

Comment:
unit tests should not contain business logic, in other words there should be no if-else etc. for this particular case it's required to split the test into 2 separate ones